docs/tutorials/reuse-ssh-connections.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Reuse SSH Connections
+group: advanced-networking
+author: [ "@Dave-Baum" ]
+---
+
+* Table of Contents
+{:toc}
+
+## Intro
+
+Establishing an ssh connection requires a a lot of communication between your host computer
+and the EV3 and can take several seconds.  Although this is fine a long-lived connection,
+it can be quite tedious if you want to use ssh to invoke a single command or scp to transfer
+files.  By instructing ssh to reuse connections, each connection after the first one will
+reuse the session that was already established, drastically speeding up ssh and scp commands.
+
+## Congifuration (OSX, Linux)
+
+Edit or create a new file at `~/.ssh/config`.  Add the following contents:
+
+    Host <name>
+      HostName <address>
+      User <user>
+      ControlMaster auto
+      ControlPath ~/.ssh/master-%r@%h:%p
+
+Where `<name>` is the name you want to use to refer to your ev3, `<address>` is its ip address, and `<user>` is
+the user name (the default is `robot`).  For example, if your ev3 has address `192.168.1.10`, you want to
+call it `ev3` and connect as user `robot`, the file would look like this:
+
+    Host ev3
+      HostName 192.168.1.10
+      User robot
+      ControlMaster auto
+      ControlPath ~/.ssh/master-%r@%h:%p
+
```

Comment:
ev3dev ships with avahi, so mDNS resolution works out of the box with macOS and Linux. I think this would be sufficient for most people:

    Host ev3dev.local.
    User robot
    ControlMaster auto
    ControlPath ~/.ssh/master-%r@%h:%p

Then you just connect with `ssh robot@ev3dev.local.`